doc/user/content/integrations/websocket-api.md

```diff
@@ -59,8 +59,9 @@ To authenticate using a username and password, send an initial text or binary me
 
 ```
 {
-  "user": "<Your email to access Materialize>"
-	"password": "<Your app password>",
+    "user": "<Your email to access Materialize>",
+    "password": "<Your app password>",
+    "options": { <Optional map of run-time settings> }
```

Comment:
By run-time settings, do we mean https://materialize.com/docs/sql/show/#session-variables? If so, we say "Optional map of session variables"? I wonder, also, if there's a way for us to mention this in the intro to this section or under General Semantics.

Reply:
Updated to say "Optional map of session variables". I'm going to further refine the docs for the websocket and http APIs in a followup PR, I'll take a pass at updating the intro section or General Semantics there :)